modules/power-monitoring-configuring-kepler-redfish.adoc

```diff
@@ -8,14 +8,19 @@
 
 You can configure {PM-kepler} to use Redfish as the source for running or hosting containers. {PM-kepler} can then monitor the power usage of these containers.
 
+[IMPORTANT]
+====
+Configuring {PM-kepler} to use Redfish applies only for the Power Monitoring Operator 0.4 and earlier.
+====
```

Comment:
We only support the latest version of OpenShift Power Monitoring. Once the 0.5 TP is released, all previous versions that include Redfish support will no longer be supported. I would recommend removing the whole Redfish section as customers will have to update to 0.5 TP. The removal of Redfish support should be mentioned in the release notes.